src/System.Net.NameResolution/ref/project.lock.json

```diff
@@ -1,5 +1,5 @@
 {
-  "locked": true,
+  "locked": false,
```

Comment:
I believe locked should be true here.  @ericstj, that's true even for the ref files, correct?